go.mod

```diff
@@ -1 +0,0 @@
-module github.com/goproxyio/goproxy
```

Comment:
why you remove this line?

Reply:
It looks like a mistake.
I recover module files.